spec.mediawiki

```diff
@@ -425,12 +403,11 @@ trade assets on chain A for assets on chain B.
 Limit orders are for the trade of assets at a rate no higher (buy) or lower 
 (sell) than a specified price. 
 The client may specify the "time in force" of a limit order as one of: (a) 
-''Standing'', which remains on the books until filled or cancelled, or (b) 
-''Immediate'', which will be cancelled if it is not filled during during epoch 
-order matching. As such, the ''Immediate'' option is intended for limit orders 
-with a price that crosses the spread (i.e. a taker rather than a maker).
-The order <code>quantity</code> must be an integer multiple of the lot size 
-(see 3.1.4). 
+''standing'', which remains on the books until filled or cancelled, or (b) 
+''immediate'', which can complete execution wholly or partially unfilled. As 
+such, the ''immediate'' option is intended for limit orders with a price that 
+crosses the spread (i.e. a taker rather than a maker). The order 
+<code>quantity</code> must be an integer multiple of the lot size (see 3.1.4). 
 
```

Comment:
A little rewrite here. I noticed it was *standing* and *immediate* were capitalized some places and lower-case in others. Hope lower-case is ok. 

Also removed the word "cancelled". In the spec, I'd like to reserve the specifier "cancelled" for standing limit orders that have matched a cancellation order only. Unfilled market orders or unfilled immediate limit orders are not cancelled, just "unfilled". This is partially because we are considering tracking cancellations, and the latter would not count as a cancellation, so using that term could get a little confusing.